graph/testdata/acb.yaml

```diff
@@ -15,7 +15,7 @@ steps:
   - id: puller
     cmd: azure/images/docker pull ubuntu
     entryPoint: someEntryPoint
-    envs: [eric=foo, foo=bar]
+    env: [eric=foo, foo=bar]
     secretEnvs: [someAkvSecretEnv]
```

Comment:
should the format of secretEnvs be same as  env?

Reply:
`secretEnvs` should probably be deleted for now until MSI work is done.